src/main/osd/osd.c

```diff
@@ -428,7 +428,13 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
     osdConfig->osd_use_quick_menu = true;
 #endif // USE_OSD_QUICK_MENU
 #ifdef USE_SPEC_PREARM_SCREEN
```

Comment:
```suggestion
#if defined(USE_SPEC_PREARM_SCREEN) && defined(USE_RACE_PRO)
osdConfig->osd_show_spec_prearm = true;
#endif
```
pg_config is designed to default to `false`/`0`/`NULL`

Reply:
I will change this

Reply:
Having the `USE_RACE_PRO` condition we no longer have to check `USE_SPEC_PREARM_SCREEN` as it is included when `USE_RACE_PRO` is defined.